mergebot_backend/merge.py

```diff
@@ -372,18 +433,25 @@ def verify_pr_via_jenkins(self, job, build_num, pr, pr_logger):
                 time.sleep(self.WAIT_INTERVAL)
                 wait_secs += self.WAIT_INTERVAL
 
-        job_url = urlparse.urljoin(
-            self.config.jenkins_location, '/job/{job_name}/'.format(
-                job_name=self.config.verification_job_name))
         if not build:
             pr_logger.error('Timed out trying to find the verification job.')
+            pr.post_commit_status(
+                state=github_helper.COMMIT_STATE_ERROR,
+                url=job_url,
+                description='Timed out waiting for job start.',
```

Comment:
There will be resource waste if timeout here since Jenkins will run the queued job anyway. Another [issue](https://github.com/jasonkuster/merge-bot/issues/23) is created and we can address it in another PR.